Interesting...
```bash
FAILED: CMakeFiles/FEBioStudio.dir/FEBioStudio/FileViewer.cpp.o
/nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/clang++ -DFFMPEG -DHAS_SSH -DHAVE_ZLIB -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_OPENGLWIDGETS_LIB -DQT_OPENGL_LIB -DQT_WIDGETS_LIB -DTETLIBRARY -D__APPLE__
-I/tmp/nix-build-febio-studio-2.7.drv-0/source/build/CMakeFiles/AutoGen/FEBioStudio_autogen/include -I/tmp/nix-build-febio-studio-2.7.drv-0/source/.
-F/nix/store/nzfcir6lrdq1dx2vkwl5xv996p60zhnq-apple-sdk-12.3/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks -isystem /nix/store/24n4sn70knrm4ma6g8j3dv4xfbq2axvj-qtbase-6.8.0/lib/QtCore.framework/Headers -iframework
/nix/store/24n4sn70knrm4ma6g8j3dv4xfbq2axvj-qtbase-6.8.0/lib -isystem /nix/store/24n4sn70knrm4ma6g8j3dv4xfbq2axvj-qtbase-6.8.0/mkspecs/macx-clang -isystem /nix/store/24n4sn70knrm4ma6g8j3dv4xfbq2axvj-qtbase-6.8.0/lib/QtWidgets.framework/Headers -isystem
/nix/store/24n4sn70knrm4ma6g8j3dv4xfbq2axvj-qtbase-6.8.0/lib/QtGui.framework/Headers -isystem /nix/store/24n4sn70knrm4ma6g8j3dv4xfbq2axvj-qtbase-6.8.0/lib/QtNetwork.framework/Headers -isystem /nix/store/24n4sn70knrm4ma6g8j3dv4xfbq2axvj-qtbase-6.8.0/lib/QtOpenGL.framework/Headers -isystem /nix/store/24n4sn70knrm4ma6g8j3dv4xfbq2axvj-qtbase-6.8.0/lib/QtOpenGLWidgets.framework/Headers -O3 -DNDEBUG -std=gnu++17 -isysroot /nix/store/nzfcir6lrdq1dx2vkwl5xv996p60zhnq-apple-sdk-12.3/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -mmacosx-version-min=10.15
-F/nix/store/6sqx120cqlp20k69fp5al6aji6rri5k0-apple-sdk-10.12.2/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks  -w -fPIC -MD -MT CMakeFiles/FEBioStudio.dir/FEBioStudio/FileViewer.cpp.o -MF CMakeFiles/FEBioStudio.dir/FEBioStudio/FileViewer.cpp.o.d -oCMakeFiles/FEBioStudio.dir/FEBioStudio/FileViewer.cpp.o -c /tmp/nix-build-febio-studio-2.7.drv-0/source/FEBioStudio/FileViewer.cpp
/tmp/nix-build-febio-studio-2.7.drv-0/source/FEBioStudio/FileViewer.cpp:651:17: error: variable has incomplete type 'QDirIterator'
                        QDirIterator it(path, { "*.fs2", "*.fsm", "*.feb", "*.xplt" }, QDir::AllEntries | QDir::NoSymLinks | QDir::NoDotAndDotDot);
                                     ^
/nix/store/24n4sn70knrm4ma6g8j3dv4xfbq2axvj-qtbase-6.8.0/lib/QtCore.framework/Headers/qdir.h:17:7: note: forward declaration of 'QDirIterator'
class QDirIterator;
      ^
1 error generated.

--- a/pkgs/by-name/fe/febio-studio/package.nix
+++ b/pkgs/by-name/fe/febio-studio/package.nix
@@ -1,7 +1,6 @@
 {
   lib,
   stdenv,
-  overrideSDK,
   fetchFromGitHub,
   cmake,
   ninja,
@@ -10,6 +9,8 @@
   qt6Packages,
   febio,
   glew,
+  apple-sdk_11,
+  darwinMinVersionHook,
   sshSupport ? true,
   openssl,
   libssh,
@@ -23,18 +24,7 @@
   withCadFeatures ? false,
 }:
 
-let
-  stdenv' =
-    if stdenv.hostPlatform.isDarwin then
-      overrideSDK stdenv {
-        darwinSdkVersion = "11.0";
-        darwinMinVersion = "10.15";
-      }
-    else
-      stdenv;
-in
-
-stdenv'.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: {
   pname = "febio-studio";
   version = "2.7";
 
@@ -76,7 +66,11 @@ stdenv'.mkDerivation (finalAttrs: {
     ]
     ++ lib.optional tetgenSupport tetgen
     ++ lib.optional ffmpegSupport ffmpeg
-    ++ lib.optional dicomSupport dcmtk;
+    ++ lib.optional dicomSupport dcmtk
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      apple-sdk_11
+      (darwinMinVersionHook "10.15")
+    ];
 
   meta = {
     description = "FEBio Suite Solver";

--- a/pkgs/by-name/fe/febio/package.nix
+++ b/pkgs/by-name/fe/febio/package.nix
@@ -1,23 +1,19 @@
 {
   lib,
   stdenv,
-  overrideSDK,
   fetchFromGitHub,
   fetchpatch2,
   substituteAll,
+  apple-sdk_11,
   cmake,
+  darwinMinVersionHook,
   ninja,
   zlib,
-  darwin,
   mklSupport ? true,
   mkl,
 }:
 
-let
-  stdenv' = if stdenv.hostPlatform.isDarwin then overrideSDK stdenv "11.0" else stdenv;
-in
-
-stdenv'.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: {
   pname = "FEBio";
   version = "4.7";
 
@@ -57,9 +53,8 @@ stdenv'.mkDerivation (finalAttrs: {
     [ zlib ]
     ++ lib.optionals mklSupport [ mkl ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
-      darwin.apple_sdk.frameworks.CoreGraphics
-      darwin.apple_sdk.frameworks.CoreVideo
-      darwin.apple_sdk.frameworks.Accelerate
+      apple-sdk_11
+      (darwinMinVersionHook "10.15")
     ];
 
   meta = {

